### PR TITLE
Feature/RC lights more names and status values

### PR DESCRIPTION
**Pull request by KhrystynaDubovyk**
Technical task:  #[2172](https://github.com/smartdevicelink/sdl_core/issues/2172)
Update Enums, Structs index with new Lights parameters.

### CODE REVIEW

**Comment by yang1070** [comment]
@theresalech Ford approves this change. thanks

File: docs/Common/Enums/index.md [hunk 1]
```diff
@@ -230,6 +230,17 @@
 |CID1SET|2||
 |CID2SET|3||
 
+### DeactivateReason
+
+|Name|Value|Description|
+|:---|:----|:----------|
+|AUDIO|0||
```
Comment (JackLivio) [inline docs/Common/Enums/index.md]: I don't believe this `DeactivateReason` is related to status light values?

File: docs/Common/Structs/index.md [hunk 2]
```diff
@@ -288,7 +288,7 @@
 |passengerBuckleBelted|Common.VehicleDataEventStatus|false|||
 |driverBuckleBelted|Common.VehicleDataEventStatus|false|||
 |leftRow2BuckleBelted|Common.VehicleDataEventStatus|false|||
-|passengerChildDetected|Common.VehicleDataEventStatus|false|||
+|passengerChildDetected|Common.VehicleVehicleDataEventStatus|false|||
```
Comment (JackLivio) [inline docs/Common/Structs/index.md]: Typo? Common.VehicleDataEventStatus?

File: docs/Common/Structs/index.md [hunk 3]
```diff
@@ -487,7 +487,6 @@
 |radioEnable|Boolean|false||True if the radio is on, false is the radio is off|
 |state|Common.RadioState|false||||
 |sisData|Common.SisData|false||Read-only Station Information Service (SIS) data provides basic information about the station such as call sign, as well as information not displayable to the consumer such as the station identification number|
-|hdRadioEnable|Boolean|false||True if the hd radio is on, false is the radio is off|
```
Comment (JackLivio) [inline docs/Common/Structs/index.md]: Why was this deleted?

File: docs/Common/Enums/index.md (hunk 1 above)
Comment (KhrystynaDubovyk) [reply docs/Common/Enums/index.md]: @JackLivio, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/107/commits/13acbe8d0475d92bbd431d2bf48dce0cb53ea485

File: docs/Common/Structs/index.md (hunk 2 above)
Comment (KhrystynaDubovyk) [reply docs/Common/Structs/index.md]: @JackLivio, fixed https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/107/commits/4b3ac0992486809d519c739047368291b25a9304

File: docs/Common/Structs/index.md (hunk 3 above)
Comment (KhrystynaDubovyk) [reply docs/Common/Structs/index.md]: @JackLivio, solving conflict I removed the parameter thinking it is not related to status light values.
after investigation, it turned out the param was added and merged by earlier #pull/106.
added back https://github.com/smartdevicelink/sdl_hmi_integration_guidelines/pull/107/commits/f917f54c779c148ec6928936f871d7b60f4d6150
